woocommerce/woocommerce-ios
# Remove OrdersVC xib

I'm eliminating possible causes of #2241. 

The stack trace of #2241 suggest that the `performFetch` was not called when the `didSelectRow` happened which seems impossible. With the (empty) xib, I feel like I have less control over when the sequence of `UITableView` initialization and `ResultsController` happen. Hopefully, creating the `UITableView` manually and combined with the changes in #2362 would help me move on and find other places that could cause the issue. 

## Changes

Along with the removal of the `OrdersViewController.xib`, I also reordered the `viewDidLoad` so that the UI initialization happens first. 

https://github.com/woocommerce/woocommerce-ios/blob/35ed00f38e8604ec1da018bf131b218e7fab6223/WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift#L108-L122

## Testing

Please do a regression test:

1. Make sure that orders are still being loaded and shown. 
2. Switch to a different store and confirm that the ghost is still being shown. 
3. Confirm that you can navigate to a specific order.

## Reviewing 

Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
